setup-client.yml

```diff
@@ -43,11 +47,39 @@
       - hibiscus-ppa
       - gnucash
       - gnucash-docs
+      - wine32
+      - winetricks
+      - playonlinux
 
-  - name: Erstelle /etc/issue
-    template:
-      src: issue
-      dest: /etc/issue
+# XnViewMP installieren
+  - name: XnViewMP manuell runterladen
+    get_url:
+      url: http://download.xnview.com/XnViewMP-linux-x64.deb
+      dest: /tmp/XnViewMP-linux-x64.deb
+  - name: XnViewMP manuell installieren
+    apt: deb="/tmp/XnViewMP-linux-x64.deb"
+  - name: deb-Paket für XnViewMP löschen
+    file:
+      path: /tmp/XnViewMP-linux-x64.deb
+      state: absent
+
+# Master-PDF-Editor installieren
+  - name: PDF-Editor manuell runterladen
+    get_url:
+      url: https://code-industry.net/public/master-pdf-editor-5.3.02_qt5.amd64.deb
```

Comment:
ggf Versionsnummer in eine Variable und dann unter group_vars/all.yml diese definieren